flow/util/genMetrics.py

```diff
@@ -491,76 +491,72 @@ def extract_metrics(cwd, platform, design, flow_variant, output, hier_json):
     metrics_dict['cts__clock__latency__max'] = latency_max
     metrics_dict['cts__clock__skew__worst'] = skew
 
-    extractTagFromFile('cts__timing__setup__tns__prerepair',
+    extractTagFromFile('cts__timing__setup__tns__pre_repair',
                        metrics_dict,
                        baseRegEx.format('cts pre-repair report_tns',
                                         'tns (\S+)'),
                        logPath + '/4_1_cts.log')
 
-    extractTagFromFile('cts__timing__slew__violation__count__prerepair',
+    extractTagFromFile('cts__timing__drv__slew__pre_repair',
                        metrics_dict,
                        baseRegEx.format(
                            'cts pre-repair max_slew_violation_count',
                            'max slew violation count (\S+)'),
                        logPath + '/4_1_cts.log')
 
-    extractTagFromFile('cts__timing__fanout__violation__count__prerepair',
+    extractTagFromFile('cts__timing__drv__max_fanout__pre_repair',
                        metrics_dict,
                        baseRegEx.format(
                            'cts pre-repair max_fanout_violation_count',
                            'max fanout violation count (\S+)'),
                        logPath + '/4_1_cts.log')
 
-    extractTagFromFile('cts__timing__max_cap__violation__count__prerepair',
+    extractTagFromFile('cts__timing__drv__max_cap__pre_repair',
                        metrics_dict,
                        baseRegEx.format(
                            'cts pre-repair max_cap_violation_count',
                            'max cap violation count (\S+)'),
                        logPath + '/4_1_cts.log')
 
-    extractTagFromFile('cts__timing__setup__ws__prerepair',
+    extractTagFromFile('cts__timing__setup__ws__pre_repair',
                        metrics_dict,
                        baseRegEx.format('cts pre-repair report_worst_slack',
                                         'worst slack (\S+)'),
                        logPath + '/4_1_cts.log')
 
-    extractTagFromFile('cts__timing__setup__tns__postrepair',
+    extractTagFromFile('cts__timing__setup__tns__post_repair',
                        metrics_dict,
                        baseRegEx.format('cts post-repair report_tns',
                                         'tns (\S+)'),
                        logPath + '/4_1_cts.log')
 
-    extractTagFromFile('cts__timing__setup__ws__postrepair',
+    extractTagFromFile('cts__timing__setup__ws__post_repair',
                        metrics_dict,
                        baseRegEx.format('cts post-repair report_worst_slack',
                                         'worst slack (\S+)'),
                        logPath + '/4_1_cts.log')
 
-    extractTagFromFile('cts__timing__slew__violation__count__postrepair',
+    extractTagFromFile('cts__timing__drv__slew__post_repair',
```

Comment:
It's a bit odd that it is drv_slew but drv_max_fanout - why does one have max and the other doesn't?

Reply:
Yes should we change this to max_tansistion.  If it is OK we can roll this in with the next set of changes

Reply:
Update genMetrics.py to use drv__max_slew  -- will update Metrics sheet in GitHub to reflect this